packages/website/components/navigation/nav-header.tsx

```diff
@@ -1,20 +1,26 @@
+import * as React from 'react';
 import {
   ContainerHeader,
   HeaderSection,
   ItemAvatar,
 } from '@atlaskit/navigation-next';
 import { gridSize as gridSizeFn } from '@atlaskit/theme';
 import titleCase from 'title-case';
-import * as PropTypes from 'prop-types';
 
 /** @jsx jsx */
+// @ts-ignore
 import { jsx } from '@emotion/core';
```

Comment:
I thought we don't use this no more

Reply:
*shrug* it's currently here and resolving that wasn't what I wanted to do this PR

Reply:
Valid. I'm still confused about when emotion 10 works and doesnt work